src/App.svelte

```diff
@@ -208,13 +191,16 @@
     margin: auto;
     height: 100%;
     width: 90%;
-    max-width: 880px;
     display: flex;
     flex-direction: column;
     align-items: center;
     justify-content: center;
   }
 
+  :global(.fullpage > *) {
```

Comment:
Ich nehme an, dass der Ansatz die max-width innerhalb des .fullpage stylings nicht mehr funktioniert. Ich würde trotzdem versuchen, das auf konventionellen Weg hinzubekommen, da einem solche Konstrukte beim Weiterentwickeln des Layouts in die Quere kommen werden